src/graph/GraphService.cpp

```diff
@@ -17,7 +18,8 @@ namespace graph {
 Status GraphService::init(std::shared_ptr<folly::IOThreadPoolExecutor> ioExecutor) {
     sessionManager_ = std::make_unique<SessionManager>();
     executionEngine_ = std::make_unique<ExecutionEngine>();
-    authenticator_ = std::make_unique<SimpleAuthenticator>();
+    // authenticator_ = std::make_unique<SimpleAuthenticator>();
+    authenticator_ = std::make_unique<LdapAuthenticator>();
```

Comment:
This is just for UT testing, you can ignore it temporarily, and wait for the user manager's pr to be merged and then modify it.